qtp_job_output_folder/tests/test_summary.py

```diff
@@ -61,36 +69,58 @@ def test_summary(self):
         # asserting content of html
         res = self.qclient.get("/qiita_db/artifacts/%s/" % aid)
         # cleaning artifact files, to avoid errors
-        [self._clean_up_files.extend([ff['filepath']])
-         for f in res['files'].values() for ff in f]
-        html_fp = res['files']['html_summary'][0]['filepath']
+        [
+            self._clean_up_files.extend([ff["filepath"]])
+            for f in res["files"].values()
+            for ff in f
+        ]
+        html_fp = res["files"]["html_summary"][0]["filepath"]
         with open(html_fp) as html_f:
             html = html_f.read()
 
-        self.assertCountEqual(
-            sorted(html.replace('<br/>', '').split('\n')),
-            sorted(EXP_HTML.format(aid=aid).replace('<br/>', '').split('\n')))
+        print("-------------")
+        print("-------------")
+        print(html)
+        print("-------------")
+        print(EXP_HTML.format(aid=aid))
+        print("-------------")
+        print("-------------")
+        self.assertEqual(html, EXP_HTML.format(aid=aid))
+
+        # verifying the new MANIFEST.txt
+        mfp = join(res["files"]["directory"][0]["filepath"], "MANIFEST.txt")
+        self.assertTrue(exists(f"{mfp}"))
+        with open(mfp, "r") as f:
+            obs = f.readlines()
+        self.assertCountEqual(obs, EXP_MANIFEST)
```

Comment:
I know that full file paths are problematic, but how about testing equality of the basename of each entry?